Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "rg"
-version = "0.2.1"
+version = "0.2.2"
```

Comment:
Please don't bump this version. I have my own release process that does this.

Reply:
Change reverted